src/Domain/Creators/SphereTimeDependentMaps.hpp

```diff
@@ -131,15 +146,24 @@ struct TimeDependentMapOptions {
     std::optional<std::variant<KerrSchildFromBoyerLindquist>> initial_values{};
   };
 
-  using options = tmpl::list<InitialTime, ShapeMapOptions>;
+  struct InitialVelocity {
+    static std::string name() { return "InitialVelocity"; }
```

Comment:
We probably also want to be able to specify the initial translation itself, not just the initial velocity, so we'll need an option for that. To keep things organized, make the translation options like the shape options. One struct that has

```cpp
struct TranslationMapOptions {
  using type = TranslationMapOptions;
  static std::string name() { return "TranslationMap"; }

  struct InitialValues {
    using type = std::array<std::array<double, 3>, 2>;
    ...
  };

  ...

  using options = tmpl::list<InitialValues>;
  std::array<std::array<double, 3>, 2> initial_values{};
};
```